dags/analysis.py

```diff
@@ -17,10 +19,47 @@
     "retry_delay": timedelta(minutes=30),
 }
 
-with DAG("public_analysis", default_args=default_args, schedule_interval="0 6 * * *") as dag:
+with DAG("public_analysis", default_args=default_args, schedule_interval="0 1 * * *") as dag:
+    # Built from https://github.com/mozilla/forecasting/tree/master/anomdtct
+    anomdtct_image = "gcr.io/moz-fx-data-forecasting/anomdtct:latest"
+
+    anomdtct = GKEPodOperator(
+        task_id="anomdtct",
+        name="anomdtct",
+        image=anomdtct_image,
+        email=["ascholtz@mozilla.com", "jmccrosky@mozilla.com",],
+        arguments=["{{ds}}"]
+        + ["--spreadsheet-id=" + Variable.get('anomdtct_spreadsheet_id')],
+        dag=dag,
+    )
+
+    wait_for_clients_daily = ExternalTaskSensor(
+        task_id="wait_for_clients_daily",
+        external_dag_id="main_summary",
+        external_task_id="clients_daily",
+        dag=dag,
+    )
+
+    wait_for_clients_first_seen = ExternalTaskSensor(
+        task_id="wait_for_clients_first_seen",
+        external_dag_id="main_summary",
+        external_task_id="clients_first_seen",
+        dag=dag,
+    )
+
+    anomdtct.set_upstream(
+        [
+            wait_for_clients_daily,
+            wait_for_clients_first_seen,
+        ]
+    )
+
     deviations = bigquery_etl_query(
         task_id="deviations",
         destination_table="deviations_v1",
         dataset_id="telemetry_derived",
+        arguments=("--replace",),
```

Comment:
I would have expected that we need to explicitly set project to shared-prod for the query. Not sure why this hasn't caused problems.